README.md

```diff
@@ -307,7 +307,14 @@ editor written by Salvatore Sanfilippo (antirez) in C, and
 * Memory safety, thanks to Rust!
 * Many bug fixes
 
-## Dependencies
+## Contributing
+
+The most important limitation is that the line check the line count remains
+under 1024. To check this run:
+
+```shell
+cargo xtask count-loc
+```
 
 This project must remain tiny, so using advanced dependencies such as [`ncurses`](https://crates.io/crates/ncurses),
```

Comment:
Actually, could you keep this section below under ## Dependencies? This was more meant as an explanation of the project's philosophy than contributing guidelines

Reply:
e16f332..f24b462